packages/lesswrong/lib/collections/posts/schema.tsx

```diff
@@ -2157,7 +2157,7 @@ const schema: SchemaType<DbPost> = {
   shareWithUsers: {
     type: Array,
     order: 15,
-    canRead: ['guests'],
+    canRead: [documentIsNotDeleted],
```

Comment:
Wait is this even the right thing? That seems too permissive.

Reply:
Without actually checking, I'd that if you make the permissions tighter here then maybe shared users can't tell that they're shared(?).

If that's not the case then it's probably a problem for another PR.